kevm-pyk/src/kevm_pyk/cli.py

```diff
@@ -155,6 +451,15 @@ def default() -> dict[str, Any]:
             'counterexample_info': False,
         }
 
+    @staticmethod
+    def from_option_string() -> dict[str, str]:
+        return {
+            'counterexample-information': 'counterexample_info',
+            'no-counterexample-information': 'no_counterexample_info',
+            'node': 'nodes',
```

Comment:
How would it look to specify a list type option like `nodes` in a TOML file? I was just a bit confused since the toml option is called `node` implying a single node when I'm imagining it would look something like `node = [1, 2, 3]`. Could you add a test for this?